### PR TITLE
Class22 45 product view page related items layout

### PR DESCRIPTION
**Pull request by ShravyaGurmitkal**
# Description

Please provide a short summary explaining what this PR is about.
Created the Related Items Layout , done styling for the items and added this container in Product View Container

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by RZhyvitskyi — APPROVED** [review]

File: packages/client/src/containers/RelatedItems/RelatedItems.Container.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      setCategoryProducts(categoryProductData);
+    };
+
+    fetchProductsByCategory();
+  }, [category]);
+
+  return (
+    <div className="related-items">
+      <h1 className="realtedItems-heading">Related Items</h1>
+      <div className="related-products">
```
Comment (shpomp) [inline packages/client/src/containers/RelatedItems/RelatedItems.Container.js:24]: As the Srpint2 is getting a bit stricter, I will ask for some renaming. 🙏🏽 
We need to make up our minds here whether it is related-items or related-products and make the class naming more unified, all the way up to the component itself too.

**Review by shpomp — CHANGES_REQUESTED** [review]
Overall very nice and clean 👍🏽 
Just some naming and some TODO process 🙏🏽

File: packages/client/src/containers/RelatedItems/RelatedItems.Container.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {categoryProducts.map((prod) => (
+          <ProductCard key={prod.id} product={prod} rating={5} />
+        ))}
```
[the remark this answers is not in the document]
Comment (shpomp) [reply packages/client/src/containers/RelatedItems/RelatedItems.Container.js:27]: https://hackyourfuture-cph.gitbook.io/final-project-boilerplate/in-depth/creating-pull-requests

File: packages/client/src/containers/RelatedItems/RelatedItems.Container.js (hunk 1 above)
Comment (ShravyaGurmitkal) [reply packages/client/src/containers/RelatedItems/RelatedItems.Container.js]: Changed the name to related-items

File: packages/client/src/containers/RelatedItems/RelatedItems.Container.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (ShravyaGurmitkal) [reply packages/client/src/containers/RelatedItems/RelatedItems.Container.js:28]: Created the Jira task for this and added TODO in my code

**Review by viedamayanti — APPROVED** [review]

**Review by shpomp — APPROVED** [review]

**Review by viedamayanti — APPROVED** [review]

**Pull request merged**